docs/sources/whatsnew/whats-new-in-v9-0.md

```diff
@@ -235,6 +235,26 @@ You can find the complete list of breaking changes in the links below. Please ch
 - https://grafana.com/docs/grafana/next/release-notes/release-notes-9-0-0-beta3/
 - https://grafana.com/docs/grafana/next/release-notes/release-notes-9-0-0
 
+### Envelope encryption enabled by default
+
+Since v8.3, a new kind of encryption called "envelope encryption" was added, for the secrets stored into the Grafana
+database (data source credentials, alerting notification channel credentials, oauth tokens, etc), behind a feature
+toggle named `envelopeEncryption`.
+
+So, now this feature toggle has been replaced in favor of `disableEnvelopeEncryption` and envelope encryption is
+the encryption mechanism used by default.
+
+Therefore, any secret created or updated with Grafana v9.0 won't be decryptable by any previous Grafana version unless the
+feature toggle `envelopeEncryption` is enabled (only since v8.3), which is something that needs to be considered in
+high availability setups, progressive rollouts or in case of need to roll back for any reason.
```

Comment:
```suggestion
Therefore, any secret created or updated in Grafana v9.0 won't be decryptable by any previous Grafana version unless the
feature toggle `envelopeEncryption` is enabled in the previous version(only since v8.3). 
This needs to be considered in high availability setups, progressive rollouts or in case of need to roll back to the previous Grafana version for any reason.
```

Reply:
I suggest splitting the sentence in half cause it seems a bit too complicated. WDYT?